src/translations/nb/privacy.json

```diff
@@ -8,6 +8,22 @@
     "paragraph0": "Personopplysningene knyttet til systembrukere vil ikke være gjenstand for automatisert saksbehandling eller profilering.",
     "paragraph1": "Åpne personopplysninger knyttet til ressurser registrert i tjenesten kan utsettes for profilering av andre tjenester som kopierer og/eller indekserer opplysninger i tjenesten, men profileres ikke som en del av tjenesten."
   },
+  "contact": {
+    "heading": "Kontakt",
+    "section0": {
+      "heading": "Behandlingsansvarlig",
+      "paragraph": "Institusjonen er behandlingsansvarlig for personopplysninger i tjenesten og er primært kontaktpunkt for brukere."
+    },
+    "section1": {
+      "heading": "Databehandler",
+      "paragraph": "Unit utvikler og forvalter tjenesten på vegne av behandlingsansvarlig, og har rollen som databehandler."
+    },
+    "section2": {
+      "intro": "Kontaktinformasjon til Unit brukerstøtte:",
+      "email": "Epost: <0><0></0></0>",
+      "telephone": "Telefon: <0><0></0></0>"
```

Comment:
hvorfor må du ha to "linker" her?

Reply:
i de andre linkene hvor linketeksten også skulle oversettes bruker vi `<a ...>(Placeholder)</a>`, hvor `(Placeholder)` ikke brukes til noe
Her vil vi ikke oversette epost/telefon siden de skal være lik for begge språk, så den innerste "linken" der henter innholdet til taggen vi sender inn fra komponenten